Representatives.md

```diff
@@ -0,0 +1,74 @@
+The jQuery Foundation invests significant resources in its efforts to drive Web standards on behalf of developers. The Standards Representatives coordinates those efforts, communicating on a regular basis with both the community and the projects hosted within the jQuery Foundation.
+
+The Standards Representatives will:
+
+- Know what Foundation projects are doing related to standards, and what departments or projects are involved to properly disseminate information and solicit feedback.
+- Act as ambassadors to the wider web development community, and provide feedback to the Foundation regarding its impact in that community.
+- State their affiliation with the jQuery Foundation, particularly in conferences or meetings where their attendance is a result of affiliation with, or funded by, the Foundation.
+- Provide a high-level summary of the meetings they attend, and distribute those reports as widely as possible. If there is no confidential information, public distribution is preferred.
+- Distribute information about upcoming standards meetings within the jQuery Foundation to ensure that they are attended by the appropriate representatives.
+- Inform the jQuery Foundation and interested projects of upcoming votes in the standards bodies, and take feedback on the appropriate way to vote on the matter.
+- Regularly provide reasonable data for expenses reimbursement regarding any available budget to attend meetings or events associated to their respective representation work.
+
+_Contributions can be found at [this document](https://github.com/jquery-foundation/standards/blob/master/Contributions.md)._
+
+## Standards team coordinator
+
+- Dave Methvin, @dmethvin
+
+## Advisory Committee Rep
+
+- Kris Borchers: @kborchers
+
+## Audio WG
+
+- Alexander Schmitz: @arschmitz
+
+## Browser Extension Community Group
+
+- Brian Kardell: @bkardell
+
+## Cascading Style Sheets (CSS) WG
+
+- Brian Kardell: @bkardell
+
+## Device and Sensors WG
+
+- Rick Waldron: @rwaldron
```

Comment:
I'm not the only one with a say in this but we definitely want more involvement in a11y efforts so definitely stay on. We should chat about what work goes on there and more than likely you can just keep you jQF hat on all the time.

I'm thinking we just remove CG involvement from this document since it will likely be harder to track that and I don't want the jQF in the way of folks participating in those at will.